test/e2e/storage/testsuites/base.go

```diff
@@ -550,6 +555,166 @@ func StartPodLogs(f *framework.Framework) func() {
 	return cancel
 }
 
+// LogOutput determines where output from CopyAllLogs goes.
+type LogOutput struct {
```

Comment:
This struct is only for this go file, so it is good to make it lowercase as `logOutput`.

Reply:
@pohly @msau42

Reply:
No, those were part of storage/testsuites. But this `LogOutput` is also used outside, so I'd prefer to keep it where it is.